src/styles/StyleUtils.ts

```diff
@@ -1179,7 +1179,7 @@ function getMentionTextColor(isOurMention: boolean): string {
 /**
  * Returns padding vertical based on number of lines
  */
-function getComposeTextAreaPadding(numberOfLines: number, isComposerFullSize: boolean): ViewStyle {
+function getComposeTextAreaPadding(numberOfLines: number, isComposerFullSize: boolean): ViewStyle & TextStyle {
```

Comment:
Since TextStyle extends ViewStyle I guess it should be enough
```suggestion
function getComposeTextAreaPadding(numberOfLines: number, isComposerFullSize: boolean): TextStyle {
```

Reply:
Right!